src/main/java/net/spy/memcached/MemcachedNodeROImpl.java

```diff
@@ -187,6 +187,11 @@ public int pendingOperationQueueSize() {
     return root.pendingOperationQueueSize();
   }
 
+  @Override
+  public boolean executeTlsHandshake() {
```

Comment:
What is this class used for? Do we even need to provide an implementation here instead of throwing `UnsupportedOperationException();` like the other methods?

Reply:
Seems like even if we don't need it it wouldn't hurt anything here anyways